src/components/AriaGroup/AriaGroup.types.ts

```diff
@@ -1,16 +1,8 @@
+import { AriaLabelingProps } from '@react-types/shared';
 import { ReactNode } from 'react';
+import { AriaLabelRequired } from 'src/utils/a11y';
```

Comment:
```suggestion
import { AriaLabelRequired } from '../../utils/a11y';
```
unfortunately my VS code does the same thing (sometimes), havn't found a solution yet